docs/mui-vale/styles/MUI/MuiBrandName.yml

```
@@ -12,6 +12,7 @@ swap:
   Material UI: Material UI
   MUI X: MUI X
   Base UI: Base UI
+  MUI Base: MUI Base
```

When we add rules like this, can we test that they work. This doesn't fail when the text is wrong (assuming I tested this correctly). It needs to be:

```suggestion
  MUI Base: MUI Base
```

Fixed in: https://github.com/mui/material-ui/pull/45140.